charts/hoprd-operator/templates/configmap-scripts.yaml

```
@@ -14,57 +14,50 @@ metadata:
     {{- .Values.operator.commonAnnotations | toYaml | nindent 4 }}
 {{- end }}
 data:
-  create-node.sh: |
+  create-identity.sh: |
     #!/usr/bin/env bash
 
     set -Eeuo pipefail
 
-    rm -rf "/app/node_secrets/${SECRET_NAME}"
+    export PATH=${PATH}:/root/.foundry/bin/
+    export RUST_BACKTRACE=full
+
     mkdir -p "/app/node_secrets/${SECRET_NAME}"
-    export HOPRD_PASSWORD=$(tr -cd '[:alnum:]' < /dev/urandom | fold -w30 | head -n1)
-    export HOPRD_API_TOKEN=$(echo -n "^"; tr -cd '[:alnum:]' < /dev/urandom | fold -w20 | head -n1| tr -d '[:space:]';echo -n "^")
-    node lib/main.cjs --init --api --data "/app/node_secrets/${SECRET_NAME}" --environment ${HOPRD_ENVIRONMENT} --identity "/app/node_secrets/${SECRET_NAME}/.hopr-identity" &
-    export HOPRD_PID=$!
-    checkHoprd="curl -s -o /dev/null -w '%{http_code}' 'http://localhost:3001/api/v2/account/addresses' -H 'x-auth-token: ${HOPRD_API_TOKEN}' -H 'Content-Type: application/json'"
-    while [[ "200" != `eval $checkHoprd` ]];  do
-      echo "Waiting for node to be running"; 
-      sleep 5; 
-    done
-    curl -s 'http://localhost:3001/api/v2/account/addresses' -H "x-auth-token: ${HOPRD_API_TOKEN}" -H "Content-Type: application/json" | jq > "/app/node_secrets/${SECRET_NAME}/addresses.json"
-    kill -9 $HOPRD_PID
-    export HOPRD_IDENTITY=$(cat "/app/node_secrets/${SECRET_NAME}/.hopr-identity")
-    jq -n \
-      --arg identity "$HOPRD_IDENTITY" \
-      --arg password "$HOPRD_PASSWORD" \
-      --arg api_token "$HOPRD_API_TOKEN" \
-      '$ARGS.named' > "/app/node_secrets/${SECRET_NAME}/${SECRET_NAME}.json"
+    export IDENTITY_PASSWORD=$(tr -cd '[:alnum:]' < /dev/urandom | fold -w30 | head -n1)
+    echo ${IDENTITY_PASSWORD} > /app/node_secrets/${SECRET_NAME}/identity.password
```

Is it needed to be seen in the command output?
```suggestion
    tr -cd '[:alnum:]' < /dev/urandom | fold -w30 | head -n1 > /app/node_secrets/${SECRET_NAME}/identity.password
```

---

I mean, this leaks identity.

---

Change reverted from some strange behaviour in tr